helper/include/odp/helper/threads.h

```diff
@@ -286,39 +276,6 @@ int odph_thread_create(odph_thread_t thread[],
  */
 int odph_thread_join(odph_thread_t thread[], int num);
 
-/**
- * Creates and launches odpthreads (as linux threads or processes)
- *
- * Creates, pins and launches threads to separate CPU's based on the cpumask.
- *
- * @param thread_tbl    Thread table
- * @param mask          CPU mask
- * @param thr_params    ODP thread parameters
- *
- * @return Number of threads created
- *
- * @deprecated Use odph_thread_create() instead.
- */
-int ODPH_DEPRECATE(odph_odpthreads_create)(
-	ODPH_DEPRECATE(odph_odpthread_t) *thread_tbl,
-	const odp_cpumask_t *mask,
-	const ODPH_DEPRECATE(odph_odpthread_params_t) *thr_params);
-
-/**
- * Waits odpthreads (as linux threads or processes) to exit.
- *
- * Returns when all odpthreads have terminated.
- *
- * @param thread_tbl    Thread table
- * @return The number of joined threads or -1 on error.
- * (error occurs if any of the start_routine return non-zero or if
- *  the thread join/process wait itself failed -e.g. as the result of a kill)
- *
- * @deprecated Use odph_thread_join() instead.
- */
-int ODPH_DEPRECATE(odph_odpthreads_join)(
```

Comment:
`odph_odpthreads_join()` is still mentioned in `odph_thread_join()` spec.

Reply:
V2.